Task-5-database-in-go/server.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+// importing modularatiy packages
+import (
+	"fmt"
+	"log"
+	handle "manav402/server/controller"
+	"net/http"
+	"manav402/server/DBhandler"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	var err error
+	m, err := godotenv.Read(".env")
```

Comment:
godotenv.Read()
why need .env two times.

change .env reading 1 time and take necessary env variables in 1 read.